piecewise_web/index.html

```diff
@@ -401,17 +406,28 @@ <h3 class="league-gothic">Thank you!</h3>
 map.scrollWheelZoom.disable();
 var control = L.control.zoom({position: 'topright'});
 map.addControl(control);
-var osmLayer = L.tileLayer('http://{s}.tile.osm.org/{z}/{x}/{y}.png', {
-	attribution: '&copy; <a href="http://osm.org/copyright">OpenStreetMap</a>' +
-		'contributors'
-});
 
+// Use Open Street Maps as a base tile layer
+// var osmLayer = L.tileLayer('http://{s}.tile.osm.org/{z}/{x}/{y}.png', {
+//	attribution: '&copy; <a href="http://osm.org/copyright">OpenStreetMap</a>' +
+//		'contributors'
+// });
```

Comment:
I assume you've left this osmLayer block in the code, but commented, to act as inline documentation for anyone else looking at this who _might_ prefer to use OSM maps tiles.  However, does it harm performance to leave the osmLayer uncommented? That is, if it's never used/called by anything, and doesn't harm performance, is it cleaner to just leave it in the code uncommented?